src/infra/main/template.yaml

```diff
@@ -1164,6 +1164,11 @@ Resources:
       TracingEnabled: true
       EndpointConfiguration:
         Type: PRIVATE
+        VpcEndpointIds:
```

Comment:
Currently there appears to be a strange deployment issue where if using mappings such as 
`- !FindInMap [ EnvConfig, !Ref Environment, OrchestrationVPCEndpointID ]
 - !FindInMap [ EnvConfig, !Ref Environment, AuthenticationVPCEndpointID ]`
where it pulls the values from the maps, it seems to throw this error:
`Template error: Condition token can only be used in Conditions block`